ffig/templates/_c.cpp.tmpl

```diff
@@ -1,3 +1,25 @@
+{%- macro method_parameter_list(method, final_comma=False) -%}
+{%- for arg in method.arguments -%}
+{{arg.type | to_c(module.name)}} {{arg.name}}
+{%- if final_comma -%}
+,
+{%- elif not loop.last -%}
+,
+{%- endif -%}
+{%- endfor -%}
+{%- endmacro -%}
+
+{%- macro method_arguments(method, final_comma=False) -%}
+{%- for arg in method.arguments -%}
+{{arg | restore_cpp_type}}
+{%- if final_comma -%}
+,
+{%- elif not loop.last -%}
+,
+{%- endif -%}
+{%- endfor -%}
+{%- endmacro -%}
+
 // This code was generated by FFIG <http://ffig.org>.
```

Comment:
I deleted my comment as I realised it was pointless. It WILL be on top in the generated code and I think i prefer to see macros first in the template. in short, LGTM